Makefile

```diff
@@ -92,7 +92,7 @@ $(SRC_DIR)/ir_load.c: $(SRC_DIR)/ir.g
 $(BUILD_DIR)/ir_fold_hash.h: $(BUILD_DIR)/gen_ir_fold_hash $(SRC_DIR)/ir_fold.h $(SRC_DIR)/ir.h
 	$(BUILD_DIR)/gen_ir_fold_hash < $(SRC_DIR)/ir_fold.h > $(BUILD_DIR)/ir_fold_hash.h
 $(BUILD_DIR)/gen_ir_fold_hash: $(SRC_DIR)/gen_ir_fold_hash.c $(SRC_DIR)/ir_strtab.c
-	$(BUILD_CC) $(CFLAGS) $(LDFALGS) -o $@ $^
+	$(CC) $(CFLAGS) $(LDFALGS) -o $@ $^
```

Comment:
BUILD_CC is necessary for cross compilation. It seems you set BUILD_CC anyway, so this change is not necessary.

Reply:
Ah thanks, that makes sense.